src/flint/gfp_elem.jl

```diff
@@ -344,6 +344,15 @@ function zero!(z::fpFieldElem)
   return fpFieldElem(UInt(0), R)
 end
 
+function one!(z::fpFieldElem)
+  R = parent(z)
+  return fpFieldElem(UInt(0), R)
```

Comment:
```suggestion
  return fpFieldElem(UInt(1), R)
```
this looks straight up wrong